docs/docs/Getting-Started.md

```diff
@@ -0,0 +1,35 @@
+---
+id: Getting-Started
+title: Getting Started
+---
+
+Hello Newcomer, here are some tips to help you get started contributing to this project.
+
+## Get to know the project
+Here's what you can do to know more about the project:
+
+- Read Documentation available on [GitHub Wiki](https://github.com/anitab-org/open-source-programs-backend/wiki) and [README](https://github.com/anitab-org/open-source-programs-backend/blob/develop/README.md).
```

Comment:
@Rahulm2310 for the above line you should write Docusaurus instead of Github Wiki for the getting started with the project, Also provide the relevant link that is for web & also the correct the link for the README it should be the web README.
Or if it is like getting started with the project requires setting up the backend first you could mention a line separately for that. What say?

Reply:
Also you could mention a line related to the video setup